fixtures/attribute-behavior/src/App.js

```diff
@@ -1399,23 +1403,112 @@ function RendererResult({version, result, defaultValue, didWarn, didError}) {
   return <div css={style}>{displayResult}</div>;
 }
 
-function Result(props) {
-  const {react15, react16, hasSameBehavior} = props;
-  const style = {position: 'absolute', width: '100%', height: '100%'};
-  if (!hasSameBehavior) {
-    style.border = '4px solid purple';
-  }
+function ResultPopover(props) {
   return (
-    <div css={style}>
-      <div css={{position: 'absolute', width: '50%', height: '100%'}}>
-        <RendererResult version={15} {...react15} />
-      </div>
+    <pre
+      css={{
+        padding: '1em',
+        width: '25em',
+      }}>
+      {JSON.stringify(
+        {
+          react15: props.react15,
+          react16: props.react16,
+          hasSameBehavior: props.hasSameBehavior,
+        },
+        null,
+        2,
+      )}
+    </pre>
+  );
+}
+
+class Result extends React.Component {
+  state = {showInfo: false};
+  onMouseEnter = () => {
+    if (this.interval) {
+      clearInterval(this.interval);
+    }
+    this._showInfo = true;
+    this.interval = setInterval(() => {
```

Comment:
Nit - we could prever `setTimeout` to `setInterval`.

Reply:
Brain fart, my bad